Dockerfile

```diff
@@ -11,6 +11,8 @@ ENV KARP_SERVER_ID 10
 ENV KARP_PASSWORD RAnDoM_max16char
 ENV KARP_UPSCRIPT /etc/ucarp/vip-up-default.sh
 ENV KARP_DOWNSCRIPT /etc/ucarp/vip-down-default.sh
+ENV KARP_DISABLE_HEALTHCHECK=
+ENV KARP_HEALTHCHECK_URL=https://localhost:6443/livez
```

Comment:
Since the container is running with [host networking](https://github.com/immanuelfodor/kube-karp/blob/master/helm/templates/daemonset.yaml#L32), this is the Kubernetes API server. If we check its health, the IP is released to another node that might hold a working API server. This is why you've introduced this PR, am I right?

If yes, the `KARP_HEALTHCHECK_URL` could be `KARP_K8S_HEALTHCHECK_URL` to be more descriptive. At first, I was thinking for a moment this is some kind of built-in Ucarp feature :)

Reply:
Yes.  I can, of course, change the environment name.  But I personally prefer `KARP_HEALTHCHECK_URL` because it might check any other Service as well.

Reply:
I can't think of any better name than `KARP_APICHECK_URL` if this is a general API health check to release the IP. I just want to separate the term _health_ from uCarp's own health - that's what caused me the confusion in the first place.